example/build.sh

```diff
@@ -2,7 +2,7 @@
 
 rm -rf dist/ && mkdir -p dist/ &&
 babel server.js -o dist/server.js &&
```

Comment:
Hrm, I see that `graphiql` is referenced via `../graphiql`. I'm not sure why this wasn't working entirely locally... but I feel it's more obvious this way.